mods/action.js

```diff
@@ -105,6 +105,24 @@ function apiError(apiErr) {
  * Action module code
  */
 function ActionService (options) {
+    // be backwards-compatible with apiURI-style configs
+    if(!options.apiRequest && options.apiURI) {
```

Comment:
Might be nice to print out a warning that 'You are using an outdated config format, please update your config ...'  so that the user could know that he needs to take an action. Otherwise LGTM